keystore/src/main/java/org/wildfly/security/keystore/KeyStoreUtil.java

```diff
@@ -125,6 +125,10 @@ public static KeyStore loadKeyStore(final Supplier<Provider[]> providers, final
         return result;
     }
 
+    public static KeyStore loadKeyStore(InputStream is, char[] password) throws IOException, KeyStoreException {
```

Comment:
Would be good to add some Javadoc for this method.

Reply:
Actually, this method assumes we are loading a PEM file as a keystore and just calls `loadPemAsKeyStore` directly with the same inputs and throws the same exceptions. Perhaps we can just make `loadPemAsKeyStore` public and add Javadoc to that method instead of introducing this new one. WDYT?